src/executors/mod.rs

```diff
@@ -7,50 +7,49 @@ use std::collections::HashMap;
 use std::rc::Rc;
 use std::sync::Arc;
 
+use actix_web::HttpResponse;
 use actix_web::{http::Method, web, HttpRequest};
 use anyhow::{bail, Result};
-use log::debug;
+use log::{debug, info};
+
 use pyo3_asyncio::TaskLocals;
 // pyO3 module
 use crate::types::PyFunction;
-use futures_util::stream::StreamExt;
+
 use pyo3::prelude::*;
 use pyo3::types::PyDict;
 
 /// @TODO make configurable
-const MAX_SIZE: usize = 10_000;
 
 pub async fn execute_middleware_function<'a>(
     function: PyFunction,
-    payload: &mut web::Payload,
+    payload: &mut Vec<u8>,
     headers: &HashMap<String, String>,
     req: &HttpRequest,
```

Comment:
We can remove the `req` param.

Reply:
I've removed the req since its is not used but would it be better to make it ignored like this `_req` for any later use?

Reply:
@Shending-Help , I have learned from my experience that it usually doesn't make sense to abstract for the future as much. If need be, we'll add the parameter again. 😄